pkgs/development/python-modules/pymd4c/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv
+, pkgs
```

please avoid bringing in an entire package set
```suggestion
, pkg-config  # pkgs
, pkgconfig   # pythonPackages
```